textile/digital_printing/report/fabric_printing_summary/fabric_printing_summary.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) 2023, ParaLogic and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe import _, scrub, unscrub
+from frappe.utils import cint, cstr, flt, getdate, add_days
+
+
+def execute(filters=None):
+	return FabricPrintingSummary(filters).run()
+
+
+class FabricPrintingSummary:
+	def __init__(self, filters=None):
+		self.filters = frappe._dict(filters or {})
+		self.filters.from_date = getdate(filters.from_date)
+		self.filters.to_date = getdate(filters.to_date)
+
+		if self.filters.from_date > self.filters.to_date:
+			frappe.throw(_("Date Range is incorrect"))
+
+	def run(self):
+		self.get_data()
+		self.prepare_data()
+		self.get_columns()
+
+		return self.columns, self.data
+
+	def get_data(self):
+		self.order_data = frappe.db.sql("""
+			SELECT pro.fabric_material,
+				COUNT(distinct pro.name) AS no_of_orders,
+				SUM(poi.stock_print_length) AS ordered_qty
+			FROM `tabPrint Order Item` poi
+			INNER JOIN `tabPrint Order` pro ON pro.name = poi.parent
+			WHERE pro.docstatus = 1
+				AND pro.transaction_date BETWEEN %(from_date)s AND %(to_date)s
+			GROUP BY pro.fabric_material
+		""", self.filters, as_dict=1)
+
+		self.fabric_received_data = frappe.db.sql("""
+			SELECT item.fabric_material, SUM(sed.transfer_qty) AS received_qty
+			FROM `tabStock Entry Detail` sed
+			INNER JOIN `tabStock Entry` se ON se.name = sed.parent
+			LEFT JOIN `tabItem` item ON item.name = sed.item_code
+			WHERE se.docstatus = 1
```

Comment:
- add `se.customer_provided = 1`
- remove `stock_entry_type = 'Customer Fabric Receipt'` (since it's an instance specific value)
- add `item.textile_item_type in ('Ready Fabric', 'Greige Fabric')`